src/types/accessors/AbstractMetabolicModel.jl

```diff
@@ -369,3 +369,33 @@ By default, it should be safe to do nothing.
 function precache!(a::AbstractMetabolicModel)::Nothing
     nothing
 end
+
+"""
+$(TYPEDSIGNATURES)
+
+Return the molar mass of translated gene with ID `gid`.
+"""
+gene_protein_molar_mass(model::AbstractMetabolicModel, gid::String)::Maybe{Float64} = nothing
+
+"""
+$(TYPEDSIGNATURES)
+
+Return the enzyme turnover number of reaction with ID `rid` for the reaction in
+the forward direction.
+"""
+reaction_kcat_forward(model::AbstractMetabolicModel, rid::String)::Maybe{Float64} = nothing
+
+"""
+$(TYPEDSIGNATURES)
+
+Return the enzyme turnover number of reaction with ID `rid` for the reaction in
+the backward direction.
+"""
+reaction_kcat_backward(model::AbstractMetabolicModel, rid::String)::Maybe{Float64} = nothing
+
+"""
+$(TYPEDSIGNATURES)
+
+Return all the [`Isozyme`](@ref)s associated with a `model` and reaction `rid`.
+"""
+reaction_isozymes(model::AbstractMetabolicModel, rid::String)::Maybe{Vector{Isozyme}} = nothing
```

Comment:
This is not really orthogonal to the reaction_kcat_forward and reaction_kcat_backward above. I'd vote for removing these two instead.

Also, any reason why the enzyme (isozyme/geneproduct) mass can't be saved directly in `struct Isozyme` and has to be separated?

In short, we should choose between having the accessors to the tiniest bits or the accessors for sensible units (enzymes, as with isozymes). I'm more in favor of the latter.

Reply:
(on the second read I think I got it -- `reaction_kcat_xxx` is the simplified view for the simplified enzyme constraints. Let's instead do a wrapper that smashes the isozymes? That could also finally kill the (nonsential) duality between gecko and smoment impls.)